internal/storage/v2/grpc/tracereader_test.go

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) 2025 The Jaeger Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package grpc
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/jaegertracing/jaeger/internal/storage/v2/api/tracestore"
+)
+
+func TestTraceReader_GetOperations(t *testing.T) {
+	tr := &TraceReader{}
+
+	require.Panics(t, func() {
```

Comment:
I think the panic will be from nil ptr dereference rather than from "not implemented"